shopfloor/actions/message.py

```diff
@@ -92,6 +92,13 @@ def dest_location_not_allowed(self):
     def need_confirmation(self):
         return {"message_type": "warning", "message": _("Are you sure?")}
 
+    def confirm_location_changed(self, from_location, to_location):
```

Comment:
I suggest passing a browse record here, so people who want to customize the message can use any value from a location.

Reply:
Yes changed